config/application.rb

```diff
@@ -35,5 +35,9 @@ class Application < Rails::Application
         end
       end
     end
+
+    config.middleware.use Flipper::Middleware::Memoizer, lambda {
```

Comment:
Yeah - this adds a cache per request so we don't keep hitting the db, https://github.com/jnunemaker/flipper/blob/master/docs/Optimization.md. So feel free to litter the code with feature.enabled?